pkgs/by-name/gp/gpu-usage-waybar/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  addDriverRunpath,
+}:
+rustPlatform.buildRustPackage rec {
```

Use `finalAttrs` pattern.

```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```